### PR TITLE
Bug/importer exporter

### PR DESCRIPTION
**Pull request by mattfeury**
Fix exporting to use a dynamically created model so that we don't include dropout or anything like that. This addresses #25 and some conversation from #34. 

NOTE: In `frozengraph` export, we no longer "clear devices". I think there's a way to do this using tf.train.saver, but I don't use frozengraphs so i don't know how to test that. opening that piece up to the community if someone has more experience there

### CODE REVIEW

**Comment by tumusudheer** [comment]
Hi @mattfeury ,

This doesn't fix the issue with frozen model right ? It fixes the issue with savemodel.  Let me know if I'm wrong.

Thanks

**Comment by mattfeury** [comment]
theoretically it should fix both, because we are no longer importing the model from the checkpoint but using the dynamically created Model from python. however, like i mentioned i have not tested frozen_model so feel free to.

**Comment by tumusudheer** [comment]
Sorry my bad. I didn't see the code in your second commit.

**Comment by emedvedev** [comment]
Looks good to me, merging. Will test #25 and #34 now, and if you can provide your reports as well, that'd be very much welcome!
